src/plugins/remove_button/plugin.ts

```diff
@@ -41,7 +42,9 @@ export default function (this: TomSelect, userOptions: RBOptions) {
 	const html =
 		'<a href="javascript:void(0)" class="' +
 		options.className +
-		'" tabindex="-1" title="' +
+		'" tabindex="' +
+		(options.tabindex !== undefined ? options.tabindex : '-1') +  // Use custom tabindex or default to -1.
```

Comment:
Sorry to be nitpicky, but can the ternary be removed now that the default option is set in the object `Object.assign({ ... }, userOptions)`?